### DISCUSSION TITLE
'Null' displayed in results after result

### DISCUSSION THREAD

**Discussion by Doingthejob** [microsoft-search/pnp-modern-search#2232] [Q&A]
I'm a newbie when it comes to sharepoint and a consultant installed the PNP webpart.  I am using the 'Edit result types' to pare down my results and only display people who have a '2' in their level field.  That works fine, but the people who have a '1' in their level field show up in results as 'Null'.  Is there a way to not display the 'Null' within the search results?
![pnp example](https://user-images.githubusercontent.com/109301049/179032171-a5a7bad9-d878-4a6b-b1d4-2b4bc11fa478.PNG)

**Comment by wobba** [comment]
I suggest you start at https://microsoft-search.github.io/pnp-modern-search/scenarios/ and also read https://docs.microsoft.com/en-us/microsoft-365/community/how-do-site-columns-become-managed-properties-thus-available-for-search
